js/filters.js

```diff
@@ -117,7 +117,9 @@ weechat.filter('DOMfilter', ['$filter', '$sce', function($filter, $sce) {
                 }
             }
             // recurse
-            if (node === undefined || node === null) return;
+            if (node === undefined || node === null) {
+                return;
+            }
```

Comment:
I don't have a *strong* preference, I just find it unnecessarily verbose and hard to read. But it assumes that everyone knows what they're doing so maybe it's best to not do it.

Would `if (...) { return; }` (in one line) be okay in your opinion? I'm not asking you to change it here but more of a general question